api/src/routers/reviews.js

```diff
@@ -0,0 +1,78 @@
+import express from "express";
+import knex from "../database_client.js";
+const router = express.Router();
+
+// router to get all the reviews
+router.get("/", async (req, res, next) => {
+  try {
+    const reviews = await knex("review");
+    res.json(reviews);
+  } catch (error) {
+    next(error);
+  }
+});
+
+// router to get a specific review
+router.get("/meal/:meal_id", async (req, res, next) => {
+  const { meal_id } = req.params;
+  try {
+    const reviews = await knex("review").where({ meal_id });
+    res.json(reviews);
+  } catch (error) {
+    next(error);
+  }
+});
+
+router.post("/", async (req, res, next) => {
+  try {
+    const newReview = req.body;
+    const [reviewId] = await knex("review").insert(newReview);
```

Comment:
not too familar with knex, but does this really only return the id?

Reply:
<img width="402" alt="image" src="https://github.com/user-attachments/assets/d7c29bef-dc34-4384-bdea-813fb466671a">